pages/tools.jsx

```diff
@@ -31,7 +31,7 @@ var ToolsColumn = React.createClass({
     description: React.PropTypes.string.isRequired,
     link: React.PropTypes.string.isRequired,
     src1x: React.PropTypes.string.isRequired,
-    src2x: React.PropTypes.string.isRequired,
+    src2x: React.PropTypes.string,
```

Comment:
We are passing this to `<Illustration>` ultimately and `src2x` is an optional prop of `<Illustration>`